### DISCUSSION TITLE
Stoped working after the lastest whatsapp web update

### DISCUSSION THREAD

**Discussion by sajir-sulaiman** [rizwansoaib/whatsapp-monitor#166] [General]
Hi Rizwan, 

I am using this Add-on for a while and it is an awesome one to track our loved ones in Whatsapp. Unfortunately from past 2 days after some Whatsapp web update, i am not getting a proper notification when the user comes online. I get the notification only when i open the Whatsapp web tab. I am currently using version 2.7. Pls do the needful and advise . Thanks for this wonderful add-on.

**Comment by rizwansoaib** [comment]
#145
